app/components/banner_component/banner_component.html.erb

```diff
@@ -0,0 +1,36 @@
+<div class="<%= color_class(@background_color) %> banner-component-wrapper">
+  <div class="govuk-width-container">
+    <div class="govuk-main-wrapper <%= govuk_wrapper_class %>">
+      <div class="banner-component govuk-!-padding-<%= @padding %> <%= color_class(@box_color) %> <%= media_side_class %>">
+
+        <div class="<%= text_column_class %>">
+          <div class="banner-component_title">
+            <h2 class="govuk-heading-m">
+              <%= @title %>
+            </h2>
+          </div>
+          <div class="banner-component_body">
+            <p class="govuk-body">
+              <%= @text %>
+            </p>
+            <% if @link %>
```

Comment:
If the "empty" state of `@link` is just `nil` then this if statement isn't really achieving anything. What do you think to getting rid of it and just having `<%= @link %> in isolation? Alternatively it could be changed to `if @link.present?` or something like that?

Or does the `<%= @link %>` part need to be smarter and output the text within a `link_to` tag?

Reply:
Thanks, this was an old bit of code that is no longer used, but I think it might be needed in the future. Have modified this take in an object instead, and now uses link_to to produce a better result